requirements-common.txt

```diff
@@ -10,7 +10,7 @@ protobuf # Required by LlamaTokenizer.
 fastapi >= 0.107.0, < 0.113.0; python_version < '3.9'
 fastapi >= 0.107.0, != 0.113.*, != 0.114.0; python_version >= '3.9'
 aiohttp
-openai >= 1.40.0 # Ensure modern openai package (ensure types module present)
+openai >= 1.45.0 # Ensure modern openai package (ensure types module present)
```

Comment:
Can you also update the comment line to reference this change?

Reply:
good point.
I have updated the comment to reflect the needed minimal version :heavy_check_mark: